feedback-module/src/App.js

```diff
@@ -10,7 +11,14 @@ function App({ pageTitle, lang, endpoint }) {
     i18n.changeLanguage(lang);
   }, [lang]);
 
-  return <Module pageTitle={pageTitle} endpoint={endpoint} />;
+  return (
+    <>
+      <Helmet>
+        <link rel="stylesheet" href="/main.css" />
```

Comment:
Do we want the link to be main.css or a reference to our AWS S3 link?

Reply:
Hmm I was thinking this too— I ended up deciding on main.css because it should be able to work with the AWS version anyway since they'll both be hosted in the same bucket, and if we ever need to test any local changes to the css in the future, we won't need to switch out the links.

Reply:
Ok got it. We should maybe wait to merge this in then until Rapi uploads the feedback-module.min file and we can test it?

Reply:
Sounds good!